packages/@okta/vuepress-site/docs/guides/submit-app/faqs/index.md

```diff
@@ -2,138 +2,98 @@
-* When defining schema for multi-valued attributes, it is considered a good practice to provide a `type` attribute that may be used for the purpose of canonicalization of values. In the schema definition for an attribute, the service provider MAY define the recommended canonical values (see [RFC 7643 Section 7](https://tools.ietf.org/html/rfc7643#section-7)).
+### SAML
 
-See [Section 2.4 of RFC 7643](https://tools.ietf.org/html/rfc7643#section-2.4) for more details.
+**Q: Does Okta support single logout or single sign-out (SAML protocol)?**
 
-**I only have one email/phone number/address in my user profile. Do I need to implement the array of emails/phone numbers/addresses?**
+Yes. For more information, see the Advanced Settings section of the [Using the App Integration Wizard: SAML App Wizard](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_App_Integration_Wizard).
 
-Yes, you must return these fields in an array, which is specified in the SCIM spec as a multi-valued attribute: [Section 2.4](https://tools.ietf.org/html/rfc7643#section-2.4).
+**Q: I'm an ISV setting up a SAML 2.0 app using the App Wizard and we have different domains for each customer. How do you manage these types of situations?**
 
-**Why doesn't Okta support DELETE /Users?**
+Currently, the App Wizard does not support custom domains, but public-facing integrations can still have this functionality. Create an app integration as you normally would for a single domain using the App Wizard. After you get to the step of submitting your app integration through the [OIN Manager](https://oinmanager.okta.com), you will be asked to fill out a form with all the relevant information.
 
-Okta users are never deleted for compliance and audit purposes. Okta user profiles are marked as "deactivated". Because of this, Okta never makes an HTTP DELETE request to a user resource on your SCIM API. Instead, Okta makes an HTTP PATCH request to set the `active` setting to `false`. You'll need to support the concept of an "active" and "inactive" user in your app.
+**Q: My app is now in the OIN, what is the user experience for a joint customer administrator that wants to set up single sign-on and provisioning for my app in the Okta interface?**
 
-**How does data validation work with SCIM provisioning? For example, if my app requires the phone number in a specific format, how do I ensure that Okta passes the attribute in that format? If a data validation error issue occurs how does error reporting work?**
+Okta creates unique SAML configuration documentation for each application in the OIN, so each will be different. For a sample configuration, see our instructions for [How to Configure SAML 2.0 for Salesforce](http://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-in-Salesforce.html). Also, you can search online for videos demonstrating a step-by-step walk through of all the steps an IT administrator would take to configure single sign-on and provisioning for an app.
```

Comment:
Okta creates unique SAML configuration documentation for each application in the OIN, so each will be different. 

The word "each" in that sentence is not clear as to what "each" means. Each data validation? Each document? For translations purposes as well, this should be spelled out.

For a sample configuration, see our instructions **on** [How to Configure SAML 2.0 for Salesforce](http://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-in-Salesforce.html). Also, you can search online for videos demonstrating a step-by-step walk through of all the steps an IT administrator would take to configure **Single Sign-On** and provisioning for an app.

Reply:
Changed the whole question around a bit to be clearer:
"**Q: My app is now in the OIN. What is the user experience for a customer or an administrator that wants to set up Single Sign-On for my app using the Okta interface?**

As the user experience is different for every app, Okta creates a unique SAML configuration document for each application in the OIN.

For a sample configuration, see our instructions on [How to Configure SAML 2.0 for Salesforce](http://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-in-Salesforce.html). Also, you can search online for videos demonstrating a step-by-step walk through of all the steps an IT administrator would take to configure Single Sign-On for an app."